contacts/contacts.service.js

```diff
@@ -0,0 +1,103 @@
+import {
+  data
+} from './data';
+import {
+  Person
+} from './entities';
+import {
+  CONTACTS_SERVICE_OBJECT_NAME,
+  UTILS,
+  MESSAGES
+} from './constants';
+import {
+  createPersonInstance,
+  localStorageService
+} from './helper';
+
+
+export class ContactsService {
+
+  constructor() {
+    this.internalArrayContacts = [];
+    this.init();
+  }
+
+  init = () => {
+    const localStorageObject = localStorageService.getFromLocalStorage(CONTACTS_SERVICE_OBJECT_NAME);
+    this.internalArrayContacts = (localStorageObject && localStorageObject.length) ? localStorageObject.map(item =>
+      createPersonInstance(item)) : data.map(item => createPersonInstance(item)); // eslint-disable-line
+  }
+
+  getPersonById = (personId) => {
+    const toReturnObject = this.internalArrayContacts.find(item => item.id === personId);
+    if (!toReturnObject) {
+      throw new Error(`${personId}${MESSAGES.DOES_NOT_EXIST}`);
+    }
+    return toReturnObject;
+  }
+
+  addPerson = (person) => {
+    if (!(person instanceof Person)) {
+      throw new Error(`${person}${MESSAGES.NOT_INSTANCE}`);
+    }
+    this.internalArrayContacts.unshift(person);
+  }
+
+  removePerson = (personId) => {
+    const index = this.internalArrayContacts.map(x => x.id).indexOf(personId);
+    if (index === UTILS.NOT_FOUND) {
+      throw new Error(`${personId}${MESSAGES.DOES_NOT_EXIST}`);
+    }
+    this.internalArrayContacts.splice(index, 1);
```

Comment:
you can have it like this
this.internalArrayContacts.splice(this.internalArrayContacts.findIndex(x => x.id === personId), 1)